cmd/zc_traverser_blob_account.go

```diff
@@ -94,7 +94,7 @@ func (t *blobAccountTraverser) traverse(preprocessor objectMorpher, processor ob
 
 	for _, v := range cList {
 		containerURL := t.accountURL.NewContainerURL(v).URL()
```

Comment:
Minor: hard coding the false here assumes we'll never implement account-level remove. If we ever might do that, maybe pass the includeBlobStubs in?